### DISCUSSION TITLE
Developing Terminal.Gui application in Rider

### DISCUSSION THREAD

**Discussion by bukowa** [tui-cs/Terminal.Gui#3041] [Q&A] [answered]
When executing Terminal.Gui application in Jetbrains Rider:
```
"C:\Program Files\JetBrains\JetBrains Rider 2023.2.3\plugins\dpa\DotFiles\JetBrains.DPA.Runner.exe" --handle=4220 --backend-pid=28000 --etw-collect-flags=67108622 --detach-event-name=dpa.detach.4220 
.../bin/Debug/net8.0/TerminalGuiBasicTemplate1.exe
```
![ssssz](https://github.com/gui-cs/Terminal.Gui/assets/38087302/b5dedd30-0493-4a22-85af-c44a47e5e67c)

Terminal does not handle mouse event and looks like it looks... how do we make it look "normal" ? :)

**Comment by bukowa** [comment, marked as answer]
Solution is to right click -> edit run configuration -> use external console.

**Reply by tznind** [reply]
Thanks for sharing this solution :) Always nice to build up knowledge base like this.
